drivers/cc1200/include/cc1200-internal.h

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright (C) 2017 TU Berlin DAI Labor
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_cc1200
+ * @{
+ *
+ * @file
+ * @brief       Data structures and variables for the cc1200 driver interface
+ *
+ * @author      Anon Mall <anon.mall@gt-arc.com>
+ */
+
+#ifndef CC1200_INTERNAL_H
+#define CC1200_INTERNAL_H
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define CC1200_RXBUF_SIZE           (2)
+#define CC1200_MAX_DATA_LENGTH      (58 + 64)
+
+#define CC1200_HEADER_LENGTH        (3)     /**< Header covers SRC, DST and
+                                                 FLAGS */
+#define CC1200_BROADCAST_ADDRESS    (0x00)  /**< CC1200 broadcast address */
+
+#define MIN_UID                     (0x01)  /**< Minimum UID of a node is
+                                                 1 */
+#define MAX_UID                     (0xFF)  /**< Maximum UID of a node is
+                                                 255 */
+
+#define MIN_CHANNR                  (0)     /**< Minimum channel number */
+#define MAX_CHANNR                  (33)    /**< Maximum channel number */
```

Comment:
If this is something device specific (which I don't believe) it should be prefixed with the device name. If not, we should consider extending the ieee802154.h file in a separate PR

Reply:
Same applies to all other defines